src/theme/sass-partials/_typography.scss

```diff
@@ -70,6 +70,7 @@
     @extend .regular-font-family;
     font-size: 23px;
     line-height: 26px;
+    color: black;
```

Comment:
why is this change from the original file required?

Reply:
without this the text displays as light grey. was discussed in the chat.
![image](https://user-images.githubusercontent.com/48550267/107629241-bb962300-6c59-11eb-8fb4-d8ff91424eb4.png)